src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -296,6 +296,14 @@ public String getMessageTimestampName() {
         return getString("message.timestamp.name");
     }
 
+    public int getMessageTimestampId() {
+        return getInt("message.timestamp.id", 1);
+    }
+
+    public String getMessageTimestampType(String defaultValue) {
```

Comment:
Can the defaultValue specified in the property file?  So far we have keep the convention to have the default value explicitly spelled out in the property file.